### DISCUSSION TITLE
Converting jng to png does not preserve transparency

### DISCUSSION THREAD

**Discussion by DuchGhast** [ImageMagick/ImageMagick#5366] [Help] [answered]
When converting from jng to png background becomes solid color.

**Comment by fmw42** [comment]
What is your IM version and platform/OS?  What was your exact command line?  Please provide an example input image and output so we can use it to test.

**Reply by DuchGhast** [reply]
$ magick -version
```
Version: ImageMagick 7.1.0-43 Q16-HDRI x86_64 20237 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenCL OpenMP(4.5)
Delegates (built-in): bzlib cairo djvu fontconfig freetype heic jbig jng jp2 jpeg jxl lcms lqr ltdl lzma openexr pangocairo png raqm rawrsvg tiff webp wmf x xml zip zlib
Compiler: gcc (12.1)
```
System: (Arch based) Garuda Linux

$ convert mark.jng mark.png

**Comment by fmw42** [comment, marked as answer]
I can confirm that the conversion works fine in IM 6.9.12.58, but fails in IM 7.1.0.43 as described above.

`convert mark.jng x.png`

![x](https://user-images.githubusercontent.com/13458218/180663583-da322d22-0d0d-4a78-8433-87f82a4cecf1.png)

`magick mark.jng x2.png`

![x2](https://user-images.githubusercontent.com/13458218/180663605-af288aa8-c13a-460a-84d8-614effbe8c85.png)

**Reply by dlemstra** [reply]
This issue has been resolved in the new release (`7.1.0-44`) that was published yesterday.
